ros2_control_demo_robot/description/rrbot/rrbot.urdf.xacro

```diff
@@ -68,7 +67,7 @@
         <geometry>
           <box size="${width} ${width} ${height2}"/>
         </geometry>
-        <material name="black"/>
+        <material name="yellow"/>
```

Comment:
@bmagyar  I propose here a new color because black on the dark-gray background (default) in rivz is really hard to see. White is also not great. What I found on the internet also color-blind persons should see the difference between orange (dark gray) and yellow (light gray).

Reply:
thanks :)